deploy/get-clowder-common-bash.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+
+WORKDIR=$(dirname "$0")
+VERSION='992e8a167dcdb96895c68cfab5f416e76caa57e6'
+FILENAME='clowder-config-main'
+FILEPATH="${WORKDIR}/${FILENAME}"
+INTEGRITY_FILE="${FILENAME}.sha256"
+URL="https://raw.githubusercontent.com/Victoremepunto/clowder-common-bash/${VERSION}/${FILENAME}"
```

Comment:
Can you moved this to the RedHatInsights organization? It can be even in this repo.

Reply:
Done: https://github.com/RedHatInsights/clowder-common-bash